src/helper.js

```diff
@@ -38,6 +40,10 @@ exports.countNestedProperties = object => {
     return propertiesCount
 }
 
+exports.getItemFromPath = (object, path) => {
```

Comment:
I change the syntax of the step, (for a path) is not mandatory anymore and if there is no path specified, it looks in the root object.